solutions/src/main/java/ru/sberbank/school/task02/Solution.java

```diff
@@ -0,0 +1,48 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class Solution {
+    public static void main(String[] args) {
+
+        CalculatorFactory calculatorFactory = new CalculatorFactory();
+        // получаем наш валютный калькулятор
+        FxConversionService calculator =
+                calculatorFactory.getFxConversionService(new ExternalQuotesServiceDemo());
+        ExtendedFxConversionService extendedCalculator =
+                calculatorFactory.getExtendedFxConversionService(new ExternalQuotesServiceDemo());
+
+        Beneficiary beneficiary = System.getenv("Beneficiary") == "CLIENT" ?
+                Beneficiary.CLIENT : Beneficiary.BANK;
+
+        try {
+            BigDecimal priceUSD = calculator.convert(ClientOperation.BUY,
```

Comment:
Abbreviation in name &#39;priceUSD&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-533838353>*